apps/src/applab/commands.js

```diff
@@ -1381,7 +1384,8 @@ applabCommands.startWebRequest = function (opts) {
   apiValidateType(opts, 'startWebRequest', 'callback', opts.func, 'function');
   var req = new XMLHttpRequest();
   req.onreadystatechange = applabCommands.onHttpRequestEvent.bind(req, opts);
-  req.open('GET', opts.url, true);
+  var url = XHR_PROXY + encodeURIComponent(opts.url.replace(' ', '%20'));
```

Comment:
@islemaster can you remind me what the problem with this line was that you discovered in another PR?